core/kazoo_number_manager/src/knm_phone_number.erl

```diff
@@ -253,14 +253,27 @@ authorized_release(PhoneNumber) ->
 -spec to_public_json(knm_phone_number()) -> wh_json:object().
 to_public_json(Number) ->
     JObj = to_json(Number),
-    ToLeak = wh_json:from_list(
-               props:filter_empty(
-                 wh_json:to_proplist(
-                   wh_json:private_fields(JObj)
-                  )
-                )
-              ),
-    wh_json:set_value(<<"_read_only">>, ToLeak, wh_json:public_fields(JObj)).
+    State = {<<"state">>, state(Number)},
+    UsedBy = {<<"used_by">>, used_by(Number)},
+    ReadOnly =
+        wh_json:from_list(
+          props:filter_empty(
+            [ {<<"created">>, wh_doc:created(JObj)}
+            , {<<"updated">>, wh_doc:modified(JObj)}
```

Comment:
@fenollp where is modified leaked as updated?  In the PR for 3.22 I accepted it but changed this to modified.  Specially because this is a new property... we have not had _read_only on the result before so why do you state it used to be updated?